Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG PYTHON_VERSION=3.11-slim
+
+FROM python:${PYTHON_VERSION} AS base
+ENV PYTHONUNBUFFERED=1
+WORKDIR /app
+COPY requirements.txt /app/
+RUN pip install --no-cache-dir -r requirements.txt
+
+FROM base AS build
+COPY . /app
+
+FROM base
+COPY --from=build /app /app
+RUN python manage.py migrate
```

Comment:
Running `python manage.py migrate` during the build stage might cause issues if the database isn't available. It's better to handle migrations in the entrypoint or command of the final stage to ensure the database is ready.